app/bundles/InstallBundle/Configurator/Form/EmailStepType.php

```diff
@@ -167,20 +169,13 @@ public function buildForm(FormBuilderInterface $builder, array $options)
 
         $builder->add(
             'mailer_auth_mode',
-            'choice',
+            ChoiceType::class,
             [
-                'choice_list' => new ChoiceList(
-                    [
-                        'plain',
-                        'login',
-                        'cram-md5',
-                    ],
-                    [
-                        'mautic.email.config.mailer_auth_mode.plain',
-                        'mautic.email.config.mailer_auth_mode.login',
-                        'mautic.email.config.mailer_auth_mode.cram-md5',
-                    ]
-                ),
+                'choices' => [
+                    'plain'    => 'mautic.email.config.mailer_auth_mode.plain',
+                    'login'    => 'mautic.email.config.mailer_auth_mode.login',
+                    'cram-md5' => 'mautic.email.config.mailer_auth_mode.cram-md5',
+                ],
```

Comment:
Note: We will have to test these selects carefully. This refactoring out of ChoiceList created bugs in the Page Builder.

Reply:
Tested again and works well